draft-johansson-ccwg-rfc8298bis-screamv2.md

```diff
@@ -170,11 +170,27 @@ SCReAM. The main differences are:
   that they are unnecessarily queued up on the sender side, but still prevent a
   large network queue.
 
-* The media bitrate calculation is dramatically changed and simplified.
+* The media bitrate calculation is dramatically changed and simplified. In practive
+  it is manifested with a relatively simple relation between the reference window and RTT.
 
 * Additional compensation is added to make SCReAMv2 handle cases such as large
   changing frame sizes.
 
+Algorithm changes since the last draft version -01 are:
+
+ * Slow down reference window growth when close the last know max is disabled
```

Comment:
This sentence needs fixing.

Reply:
@IngJohEricsson you will have to take a look and clarify the meaning.

Reply:
At minimum there are some nits but this sentence is not super clear.